p3/models.py

```diff
@@ -95,6 +100,12 @@ class TicketConference(models.Model):
     def __str__(self):
         return "for ticket: %s" % self.ticket
 
+    def save(self, *args, **kwargs):
+        if self.assigned_to.strip():
+            self.assigned_to = self.assigned_to.lower().strip()
```

Comment:
This is the actual logic that targets the ticket 😆 
The rest is cleanups and optimizations.